backend-rental/AppRental/Model/Offer.cs

```diff
@@ -0,0 +1,10 @@
+namespace AppRental.Model
+{
+    public class Offer
+    {
+        public int Id { get; set; }
+        public int CarId { get; set; }
```

Comment:
Here we can also add `Car` field which will be fetched based on `CarId`

Check: https://learn.microsoft.com/en-us/ef/ef6/fundamentals/relationships

We could also probably remove `CarId` and just stay with `Car` - the same as in this example: https://github.com/sulmar/usuconcept-zus-efcore8/blob/master/src/CodeFirst/Model/Order.cs , though in the microsoft docs there is "It is recommended to include properties in the model that map to foreign keys in the database."